soc/arm/st_stm32/stm32wb/power.c

```diff
@@ -58,47 +55,92 @@ __weak void pm_power_state_set(struct pm_state_info info)
 		/* The switch on HSI before entering Stop Mode is required */
 		switch_on_hsi();
 	}
+}
+
+#ifdef CONFIG_BT
```

Comment:
If you use `IS_ENABLED()` below, you do not need this conditional macro and this will be dropped by the linker.